.github/workflows/test.yml

```diff
@@ -1,3 +1,6 @@
+#We use && for all multi-line runs because Windows won't fail in all scenarios
```

Comment:
I don't understand this explanation. Why does using `&&` have an impact on Windows? Previously, if a command failed, that job would immediately exit without executing the lines that followed the failing line. Wouldn't the behaviour be the same with `&&`, regardless of if it's running on Windows?

Reply:
https://github.com/meltano/meltano/issues/6540 has the information (1 line down!) Not sure how much to get into it all in the comment? Maybe it should be very detailed, posted this mainly for this reason as it's a fix but not an amazing one :/

Reply:
@WillDaSilva  Dug a little more, there is a fix on Windows!!! Will have to look into how to implement this but if it works as I hope it does we should be able to do this and move on!
https://github.com/PowerShell/PowerShell/pull/15897

Reply:
I'll just close this PR for now!